src/components/Restaurant.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Order from "./Order";
+
+const Restaurant = () => {
+  return (
+    <div className="cont-res">
```

Comment:
What does `cont-res` mean? Beware over shortened classNames like this - sometimes difficult to understand when you're reading code.